recipe/meta.yaml

```
@@ -26,15 +26,19 @@ requirements:
     - python
     - pip
     - setuptools
+    - packaging
   run:
     - python
+    - aiohttp >=3.9.3
     - applaunchservices >=0.3.0  # [osx]
+    - asyncssh >=2.14.0,<3.0.0
     - atomicwrites >=1.2.0
     - chardet >=2.0.0
     - cloudpickle >=0.5.0
     - cookiecutter >=1.6.0
     - diff-match-patch >=20181111
-    - importlib-metadata >=4.6.0
+    - fcitx-qt5 >=1.2.7  # [linux64]
+    - fzf >=0.42.0
```

```suggestion
    - fzf >=0.42.0
    - importlib-metadata >=4.6.0
```

I think we need to restore this dependency because it's required since 5.5.6